celavi/component.py

```diff
@@ -216,24 +251,34 @@ def eol_process(self, env):
         """
         while True:
             if self.pathway:
-                # Update the component's process queue (EOL pathway) to remove the current
-                # location etc.
+                # Use the component's process queue (EOL pathway) to identify the
+                # component's next step
                 location, lifespan, distance, route_id = self.pathway.popleft()
                 factype = location.split("_")[0]
 
-                # If the component is now at a facility type that incurs material losses,
+                # If the next step for the component involves material losses,
```

Comment:
Not sure I follow starting from here. Implementing the closed loop pathways required to change how losses are handled?

Reply:
Just went back to #240 and saw the explanation...